### DISCUSSION TITLE
Ask for Input: Prioritize our next milestones

### DISCUSSION THREAD

**Discussion by aronmolnar** [Syslifters/sysreptor#88] [Announcements]
There is a lot more to do to make pentest reporting easier.
Do you have further ideas? Let us know here: https://github.com/Syslifters/sysreptor/issues/89

**Comment by MWedl** [comment]
Thanks for voting and helping us prioritize the next big features. 

The winners are Document Sharing via SysReptor (#122) and Concurrent editing of findings (#123). We plan to implement these features in H1 2024. See also our roadmap for the next months (#184).
